Tests/Resources/ViewsWithBackgroundColor.xib

```diff
@@ -19,11 +17,19 @@
             <viewLayoutGuide key="safeArea" id="X6C-YP-6rn"/>
             <point key="canvasLocation" x="84" y="76"/>
         </view>
+        <view contentMode="scaleToFill" id="Qov-nb-VBj" userLabel="View with System Color">
+            <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+            <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+            <color key="backgroundColor" systemColor="systemOrangeColor" red="1" green="0.58431372550000005" blue="0.0" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
```

Comment:
This `color` element seems old form 👀  Are you using old form intentionally?

Reply:
Yes. I didn't know which version was supported in this library.
So I used Xcode 11 which is the first introduced version of systemColor.
Should I use the latest version?